Update README dev setup to use a new directory for testing the templates. #33 (#37)
CSE-25/quick_start_express: README.md

### Run the package.

> To test the CLI tool locally, you need to link the package. Use a separate testing directory to avoid modifying files in the main quick_start_express directory.

Reviewer: ```suggestion
To test the CLI tool locally, you need to link the package. Use a separate testing directory to avoid modifying files in the root directory of `quick_start_express`.
```

> 1. **Link the Package in the Main Directory**: In the quick_start_express directory, run:

Reviewer: ```suggestion
1. **Link the Package in the Main Directory**: In the `quick_start_express` root directory, run:
```

   ```bash
   npm link
   ```

> 2. **Create a Testing Directory**: Outside the quick_start_express directory, create a testing directory:

Reviewer: Why not create a directory called `qse-test` within the root directory of `quick_start_express`. If we move out of scope, it might become hard to track files.

   ```bash
   mkdir ../qse-test
   cd ../qse-test
   ```

3. **Link qse in the Testing Directory**: In the testing directory, run:

   ```bash
   npm link qse
   ```
   
4. **Run qse Commands**: Now, you can execute any qse commands in the testing directory, such as:

   ```bash
   qse init
   qse clear
   qse -v
   ```

> **Note**: Running npm link qse in the main directory may modify package.json. Always use a separate testing directory to avoid this.

Reviewer: ```suggestion
> [!Note] 
> Running npm link qse in the main directory may modify `package.json`. Always use a separate testing directory to avoid this.
```

Refer https://github.com/orgs/community/discussions/16925 to use GitHub `md` styles like these.

### Run the package.

To test the CLI tool locally, you need to link the package. Use a separate testing directory to avoid modifying files in the root directory of `quick_start_express`.

1. **Link the Package in the Main Directory**: In the `quick_start_express` root directory, run:

   ```bash
   npm link
   ```

2. **Create a Testing Directory**: In the quick_start_express root directory, create a qse-test directory:

   ```bash
   mkdir qse-test
   cd qse-test
   ```

> 3. **Link qse in the Testing Directory**: In the testing directory, run:

Reviewer: ```suggestion
3. **Link `qse` in the Testing Directory**: In the testing directory, run:
```

   ```bash
   npm link qse
   ```
   

> 4. **Run qse Commands**: Now, you can execute any qse commands in the testing directory, such as:

Reviewer: ```suggestion
4. **Run `qse` Commands**: Now, you can execute any `qse` commands in the testing directory, such as:
```

   ```bash
   qse init
   qse clear
   qse -v
   ```

> [!Note] 

> > Running npm link qse in the main directory may modify `package.json`. Always use a separate testing directory to avoid this.

Reviewer: ```suggestion
> Running `npm link qse` in the root directory may modify `package.json`. Always use a separate testing directory to avoid this.
```
